src/ui/components/atoms/label-date/LabelDate.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { StoryObj, Meta } from '@storybook/react';
+import '../../../../globals/_variables.scss';
+import LabelDate from './LabelDate';
+
+const meta: Meta<typeof LabelDate> = {
+  title: 'ui/components/atoms/label-date',
+  component: LabelDate,
+};
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Primary: Story = {
+  args: {
+    month: 'april',
+    day: '15',
+    year: '2024',
+    color: 'primary',
+  },
+};
+
+export const Secundary: Story = {
+  args: {
+    month: 'may',
+    day: '30',
+    year: '2015',
+    color: 'secondary',
+  },
+};
+
+export const Tertiary: Story = {
+  args: {
+    month: 'July',
+    day: '10',
+    year: '2010',
+    color: 'tertiary',
```

Comment:
The component should receive the date of type Date, not parsed example
2024-06-19T18:44:18.845Z This data comes from strapi in ISO 8601, and you need to create a parser inside the component to change to the format wished